Python/pytest-replays/test_main.py

```diff
@@ -3,12 +3,17 @@
 from main import run
 
 test_cases = [
-    "How do I sign up?",
-    "How many pricing plans do you have?",
-    "What is your refund policy?",
+    "Where can I find your privacy policy?",
```

Comment:
@adamnolte decided to just use different test cases in the pytest one so that there's no conflict